server/server_test.go

```diff
@@ -240,11 +243,10 @@ func TestGzip(t *testing.T) {
 // TestMultiRangeScanDeleteRange tests that commands that commands which access
 // multiple ranges are carried out properly.
 func TestMultiRangeScanDeleteRange(t *testing.T) {
-	ts := StartTestServer(t)
-	tds := kv.NewTxnCoordSender(kv.NewDistSender(ts.Gossip()), ts.Clock())
+	tds := kv.NewTxnCoordSender(kv.NewDistSender(s.Gossip()), s.Clock(), testContext.Linearizable)
 	defer tds.Close()
 
-	if err := ts.node.db.Call(proto.AdminSplit,
+	if err := s.node.db.Call(proto.AdminSplit,
```

Comment:
I'd much rather see this outdated test file (was one of first written), create a new server in every test instead of relying on the initialization of a server in a prior test.